src/features/smartSearch/l10n/messageIds.ts

```diff
@@ -599,7 +599,25 @@ export default makeMessages('feat.smartSearch', {
   readOnly: m(
     'This Smart Search query is in read-only mode and cannot be edited.'
   ),
+  resultHint: {
+    countLabel: m<{ count: number }>(
+      '{count, plural, =1 {one person} other {# people}}'
+    ),
+    hint: m<{ count: ReactElement }>(
+      'This Smart Search will currently return {count} for you.'
+    ),
+  },
   smartSearch: m('Smart Search'),
+  statsPopper: {
+    change: m('change'),
+    headline: m('Selection impact'),
+    info: m(
+      'This is a dynamic selection and the numbers may change over time.'
+    ),
+    input: m('input'),
+    matches: m('found'),
+    output: m('output'),
+  },
```

Comment:
The wording of these and their placement, should be changed in accordance with the suggestion you had at kika&käka this morning I think.

Reply:
Done!